apps/system/js/system_window.js

```diff
@@ -0,0 +1,51 @@
+/* global AudioChannelController */
+'use strict';
+
+(function(exports) {
```

Comment:
Please add comment here to describe what this is used for.
// SystemWindow is a placeholder for the system app's mozbrowser iframe but with no actual UI. We will use mozChromeEvent/mozContentEvent to communicate with shell.js to use the actual mozbrowser iframe of the system app.